compiler-rt/lib/sanitizer_common/symbolizer/sanitizer_symbolize.cpp

```diff
@@ -77,13 +77,16 @@ bool __sanitizer_symbolize_code(const char *ModuleName, uint64_t ModuleOffset,
       auto ResOrErr = getDefaultSymbolizer()->symbolizeInlinedCode(
           ModuleName,
           {ModuleOffset, llvm::object::SectionedAddress::UndefSection});
-      Printer->print(Request,
-                     ResOrErr ? ResOrErr.get() : llvm::DIInliningInfo());
+      Printer->print(Request, ResOrErr.get());
```

Comment:
Is there a reason for the differing behavior between here and lines 89/113? Here, it will unconditionally print the value of ResOrErr.get(), while in lines 89/113, it is only printed if successful.

Reply:
typo :)